handler.go

```diff
@@ -566,33 +606,37 @@ func (fm *Frontman) processInput(input *Input, resultsChan chan<- Result) {
 			res.Check = check.Check
 
 			if check.Check.Connect == "" {
-				log.Errorf("serviceCheck: missing data.connect key")
+				checkLog.Errorf("serviceCheck: missing data.connect key")
 				res.Message = "Missing data.connect key"
+			} else if result, err := fm.runServiceCheck(check); err != nil {
+				res.Message = err.Error()
 			} else {
-				var err error
-				res.Measurements, err = fm.runServiceCheck(check)
-				if err != nil {
-					res.Message = err.Error()
-				} else {
-					succeed++
-				}
+				res.Measurements = result
+				succeed++
 			}
-
 			resultsChan <- res
 		}(check)
 	}
 
+	seenWebChecks := make(map[string]struct{}, len(input.WebChecks))
+
 	for _, check := range input.WebChecks {
+		if check.UUID == "" {
+			log.Errorf("webCheck: missing checkUuid key")
+			continue
+		} else if _, ok := seenWebChecks[check.UUID]; ok {
```

Comment:
Can be simplified by removing the `else`

Reply:
And what profit does that change bring to the table?

Reply:
same issue to `else` statement

Reply:
Fixed